### PR TITLE
update tests to 64gaslimit

### PR DESCRIPTION
**Pull request by winsvega**

### CODE REVIEW

File: test/libethereum/StateTestsFiller/Homestead/stCallCreateCallCodeTestFiller.json [hunk 1]
```diff
@@ -192,7 +206,8 @@
         "env" : {
             "currentCoinbase" : "b94f5374fce5edbc8e2a8697c15331677e6ebf0b",
             "currentDifficulty" : "45678256",
-            "currentGasLimit" : "0xffffffffffffffffffffffffffffffff",
+	    "currentGasLimit" : "2^63-1",
```
Comment (chriseth) [inline test/libethereum/StateTestsFiller/Homestead/stCallCreateCallCodeTestFiller.json:209]: This should probably be removed

**Comment by chriseth** [comment]
Tests fail, is that expected?

**Comment by winsvega** [comment]
no. 
can't get to the test log. error 404

File: test/libethereum/StateTestsFiller/Homestead/stCallCreateCallCodeTestFiller.json (hunk 1 above)
Comment (winsvega) [inline test/libethereum/StateTestsFiller/Homestead/stCallCreateCallCodeTestFiller.json:209]: why? it's just a comment

**Comment by chriseth** [comment]
Tests locally went fine.

**Pull request merged**